packages/vsce/src/extension.ts

```diff
@@ -34,7 +34,9 @@ function registerCommands(context: vscode.ExtensionContext): vscode.Disposable[]
                 },
             );
             SshConfigUtils.showSessionInTree(profile.name!, true);
-            await Gui.showMessage(`Installed Zowe SSH server on ${profile.name}`);
+            await Gui.showMessage(
+                `Installed Zowe SSH server on ${profile.profile!.host ? profile.profile!.host : profile.name}`,
```

Comment:
Nitpicky suggestion 😋 

```suggestion
                `Installed Zowe SSH server on ${profile.profile!.host ?? profile.name}`,
```

Also, are we sure that `profile.profile` will always be set?
I guess I' always a bit hesitant to use non-null assertion operators in general (`object.property!`), and I believe that's where we tend to miss a few bugs that could be getting caught at compile time 😋 

TL;DR: should we do this instead?
```suggestion
                `Installed Zowe SSH server on ${profile.profile?.host ?? profile.name}`,
```